src/main/java/com/mongodb/hibernate/internal/type/ValueConverter.java

```diff
@@ -40,8 +40,8 @@
  * implementation uses under the hood and rarely exposes, and domain values we usually use when setting parameter values
  * on our {@link PreparedStatement}, or retrieving column values from a {@link ResultSet}.
  */
-public final class ValueConversions {
-    private ValueConversions() {}
+public final class ValueConverter {
```

Comment:
I prefer the original `ValueConversions` for it is a util class with only static methods. `ValueConverter` implies it is a non-util class and requires creating an object to go about converions.

but anyway, I am not picky and what really matters is it is not hard to understand its purpose.

Reply:
I believe, this response belongs to https://github.com/mongodb/mongo-hibernate/pull/89#discussion_r2079119461.